Protos/swift-options.proto

```diff
@@ -3,7 +3,9 @@ syntax="proto2";
 
```

Comment:
Actually, https://github.com/search?l=Protocol+Buffer&q=apple_swift_prefix&ref=searchresults&type=Code&utf8=%E2%9C%93 seems to say no one but us is using the old option, so I'll update again to kill it outright.